dataland-frontend/tests/component/components/pages/LandingPage.cy.ts

```diff
@@ -0,0 +1,126 @@
+import { checkButton, checkImage, checkLinkByContent, checkLinkByTarget } from "@ct/testUtils/ExistenceChecks";
+import NewLandingPage from "@/components/pages/NewLandingPage.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+
+describe("Component test for the landing page", () => {
+  it("Check if essential elements are present", () => {
+    cy.mountWithPlugins(NewLandingPage, {
+      keycloak: minimalKeycloakMock({
+        authenticated: false,
+      }),
+    }).then(() => {
+      validateTopBar();
+      validateIntroSection();
+      validateBrandsSection();
+      // TODO test I AM INTERESTED button
+      // TODO test get in contact button
```

Comment:
Reminder to take care of todos

Reply:
Removed Todos because I added an entry to the manual_maintenance.md